Feature/demand by geography

Thomas-Sadler opened the pull request:
1) created function calculate_demand_by_geography
2) created test

Thomas-Sadler committed:
create function for LA

Calculate_demand_by_geography function
sum population over ages
only for LA, NOT BY WARD

Thomas-Sadler committed:
add by ward

Now done by ward as well as by LA
uses dplyr functions

Thomas-Sadler committed:
tidy up

lower case name conventions

Thomas-Sadler committed:
test number of wards

test = TRUE

Thomas-Sadler committed:
test: LA data = ward data by LA?

Thomas-Sadler committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into feature/demand-by-geography

# Conflicts:
#	src/r/model.R

seb231 commented on src/r/model.R:
```diff
@@ -48,13 +49,35 @@ calculate_capacity_by_geography <- function(geography){
+  #aggregated_population <- setnames(aggregated_population,"x","Demand")
+  if (geography == "ward") {
+    aggregated_population <- GM_Primary_school_population_by_area_and_age %>% 
+      group_by(Ward.Code, Ward.Name, Local.Authority) %>%
```
The calculation on demand runs on LA or ward name while calculating for capacity runs on the ward or LA code. This needs to be rectified before we are able to combine the data in the next step. Need to decide whether codes or names work better

Thomas-Sadler committed:
add LA codes

addressing Seb's comments
Get LA geo codes from shapefile and leftjoin to population data
